object-construction-checker/src/main/java/org/checkerframework/checker/objectconstruction/MustCallInvokedChecker.java

```diff
@@ -185,12 +185,15 @@ private void handleInvocation(Set<ImmutableSet<LocalVarWithTree>> defs, Node nod
                 ResetMustCall.class)
             != null) {
       checkResetMustCallInvocation(defs, (MethodInvocationNode) node);
-      incrementNumMustCall();
```

Comment:
This doesn't match my intuition - I would think that calling a reset-must-call method should create a new obligation. But we could also view it as part of the old obligation - this is really a question of how we'd like it to count.

I'm fine with either way, but I'd prefer we increment the count here.